guides/Step By Step Guide/TableOfContents.md

```diff
@@ -0,0 +1,56 @@
+Welcome to the Klasa Step by Step Guide! This guide is intended specifically for beginner developers in order to take you from learning about Klasa to being able to use all the other pages on this website with ease. The following is a list of things that this guide will cover:
+
+1. {@tutorial CreatingTheBot Understanding Klasa And Creating A Bot}
+	- What is Klasa?
+	- Why should you use Klasa?
+	- How do I start using Klasa?
+	- How to create the bot?
+2. Customizing The Bot Options
```

Comment:
If you're using them as indexed list, make sure they are all starting with `1.` 

(This answer is based on a previous response by AoDude)